src/foam/nanos/notification/email/EmailMessage.js

```diff
@@ -128,7 +128,7 @@ foam.CLASS({
       section: 'emailInformation',
       order: 100,
       gridColumns: 6,
-      tableWidth: 100
+      tableWidth: 130
```

Comment:
question : EnumTableCellFormatterRefinement's tableWidth already setting 130 as default but still needs tableWidth here?

Reply:
Yes, you are right. This is also 'Enum'. I don't need to add here. The width is the same as the default one.